UI/window-basic-settings.cpp

```diff
@@ -243,6 +243,9 @@ static void PopulateAACBitrates(initializer_list<QComboBox*> boxes)
 	}
 }
 
+void restrictresetBitrates(initializer_list<QComboBox*> boxes,
```

Comment:
Proper camelCase would be restrictResetBitrates.  If you change it, make sure you change it in all instances of the name.